docs/alerts/monitors/alert-grouping.md

```diff
@@ -37,7 +37,7 @@ Alert Grouping works for both Logs and Metrics Monitors.
 3. Select **Logs** as the type of Monitor.
 4. Enter your logs query, then select your desired Alert Grouping option:
    * **One alert per monitor**: Choose this option if you want to only receive a single alert for the entire monitor.
-   * **One alert per [group]**: Allows you to receive one notification per each unique value of the grouping field(s). You can pick more than one field for the grouping condition. In the example below, you would receive one alert for each `service` that has error count greater than 50. The input field has an auto-completion dropdown that allows you to select all the applicable fields from your query.<br/><img src={useBaseUrl('img/monitors/setup-logs.png')} alt="alert-grouping" />
+   * **One alert per [group]**: Allows you to receive one notification per each unique value of the grouping field(s). You can pick more than one field for the grouping condition. In the example below, you would receive one alert for each `service` that has error count greater than 50. The input field has an auto-completion dropdown that allows you to select all the applicable fields from your query.<br/><img src={useBaseUrl('img/monitors/setup-logs.png')} alt="alert-grouping" style={{border: '1px solid black'}} width="800" />
```

Comment:
The `style={{border: '1px solid black'}}` just need to be changed to `style={{border: '1px solid gray'}}`. You could do a global find all > replace in VS Code.

Reply:
Good catch, @kimsauce. Done. See commit "[Add gray borders to images](https://github.com/SumoLogic/sumologic-documentation/pull/3397/commits/e7a5b4069fe5d2a865d1069381bb424d13b0884a)".

Reply:
thank you! For some reason it looks like the black wasn't replaced with gray. I took care of it [here](https://github.com/SumoLogic/sumologic-documentation/pull/3397/commits/8f5a0ae3dc525100036517d37d935806c0a52f8d)!